test/runtime/runtime-python/test/main.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import json
+
+class Address:
+    def __init__(self, street_name, house_number, marriage, members, array_type, nested_object):
+        self.street_name = street_name
+        self.house_number = house_number
+        self.marriage = marriage
+        self.members = members
+        self.array_type = array_type
+        self.nested_object = nested_object
+
+    def to_json(self):
+        return json.dumps(self.__dict__)    
+    
+class NestedObject:
+    def __init__(self, test):
+        self.test = test
```

Comment:
You need to remove this definition.

It should be generated models used here instead. I.e. what you generate with https://github.com/asyncapi/modelina/blob/57787cb77068c3da92f03f5bb558c5c5d74b3d1a/test/runtime/runtime-python.ts

Reply:
@jonaslagoni Oh, sorry my bad. Fixed this now